attributes/default.rb

```diff
@@ -7,7 +7,7 @@
   shell = '/bin/false'
   homedir = '/var/lib/redis'
   package_name = 'redis-server'
-when 'centos', 'redhat', 'scientific', 'amazon', 'suse', 'fedora'
+when 'centos', 'redhat', 'scientific', 'amazon', 'suse', 'fedora', 'rocky'
```

Comment:
If this whole check was switched to platform_family things like AlmaLinux, Oracle Linux, and Rocky Linux would just work out of the box.